### PR TITLE
Add rpc port to readme

### PR DESCRIPTION
**Pull request by MakMuftic**
**Update README file with an explicit definition of RPC port**

### PR Checklist

- [x] I have run linter localy
- [x] I have run unit and integration tests locally
- [x] Rebased to master branch / merged master

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ For example `./vedran-linux-amd64 start --auth-secret=supersecret`.
 
 You can always run vedran with `--help` flag for list of all commands `vedran --help` or for list of all options for specific command `vedran start --help`.
 
+**Load balancer will expose Polkadot RPC API on port 4000 by default (can be changed using flag `--server-port`)**
```
Comment (MakMuftic) [inline README.md:50]: @mpetrunic should I expand on this, or is this enough.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ _Vedran loadbalancer_ is used in conjunction with [Vedran daemon](https://github
 
 **Run demo with `docker-compose up`**
 
+_After all components have been started, you can invoke RPC methods on `localhost:4000`_
```
Comment (MakMuftic) [inline README.md]: @mpetrun5 @mpetrunic

**Review by mpetrun5 — DISMISSED** [review]

**Review by mpetrun5 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -18,6 +18,8 @@ _Vedran loadbalancer_ is used in conjunction with [Vedran daemon](https://github
 
 **Run demo with `docker-compose up`**
 
+_After all components have been started and node has sent first valid metrics report (after 30 seconds), you can invoke RPC methods on `localhost:4000`_
```
Comment (MakMuftic) [inline README.md:21]: @mpetrun5 @mpetrunic

**Review by mpetrunic — APPROVED** [review]

**Comment by mpetrunic** [comment]
@MakMuftic test failing

**Pull request merged**